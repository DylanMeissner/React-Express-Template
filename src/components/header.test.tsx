import * as React from "react";
import Header from "./header";
import { render, screen } from "@testing-library/react";

describe("Header Component", () => {
  it("Renders the expected header content", () => {
    const { baseElement } = render(<Header title="This is a header"></Header>);

    expect(screen.getByText("This is a header")).toBeDefined();
    expect(baseElement).toMatchSnapshot();
  });
});
