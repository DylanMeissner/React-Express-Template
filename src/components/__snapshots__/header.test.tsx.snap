// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component Renders the expected header content 1`] = `
<body>
  <div>
    <h1>
      This is a header
    </h1>
  </div>
</body>
`;
